.github/workflows/playwright.yml

```diff
@@ -0,0 +1,41 @@
+name: Playwright Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-playwright:
+    timeout-minutes: 60
```

Comment:
I think the timeout was this high because in ory/docs it runs a lot of examples and docker composes which take long sometimes. 
I will commit this but we can adjust down the line.

Reply:
My bad, I mistook this as an internal timeout it Playwright. This is the timeout for the entire test. You are correct in setting it to something higher. The current value of 10 minutes makes sense.